engine/dlib/src/dmsdk/dlib/configfile_gen.hpp

```diff
@@ -1,5 +1,5 @@
 // Generated, do not edit!
-// Generated with cwd=/Users/mawe/work/defold/engine/dlib and cmd=/Users/mawe/work/defold/scripts/dmsdk/gen_sdk.py -i /Users/mawe/work/defold/engine/dlib/sdk_gen.json
+// Generated with cwd=/Users/mathiaswesterdahl/work/defold/engine/dlib and cmd=/Users/mathiaswesterdahl/work/defold/scripts/dmsdk/gen_sdk.py -i /Users/mathiaswesterdahl/work/defold/engine/dlib/sdk_gen.json
```

Comment:
Do we want to push it?

Reply:
mawe or full name doesn't matter to me imho.
It's documentation for the ones wanting to regenerate it themselves.

As for excluding only that change, it was just too tedious to remove from the PR